public/app/plugins/datasource/graphite/module.ts

```diff
@@ -10,4 +11,5 @@ class AnnotationsQueryCtrl {
 export const plugin = new DataSourcePlugin(GraphiteDatasource)
   .setQueryCtrl(GraphiteQueryCtrl)
   .setConfigEditor(ConfigEditor)
+  .setMetadataInspector(MetricTankMetaInspector)
```

Comment:
Nit: `setMetadataInspector` -> `setQueryMetadataInspector` ?

Reply:
hymm -- it is not necessarily query metadata, it is metadata atached to the response by the datasource, so maybe `setResponseMetadataInspector` but since it is in the datasource scope just `MetadataInspector` feels a bit better.

I can be convinced though ;)